doc/components_2.md

```diff
@@ -25,5 +25,5 @@ You can wrap your external mjml files inside the default `mjml > mj-body`
 </mjml>
 ```
 
-The `MJML` engine will then replace your included files before starting
-  the rendering process.
+The `MJML` engine will then include these files
+  in the HTML file as it builds the HTML.
```

Comment:
I don't think this is really better as it's replaced way before it "builds" the HTML

Reply:
Good. A fix to that makes the whole thing even better.

I think the biggest point is that the MJML engine doesn't really touch the rendering process. That happens long after MJML completes.